"flutter build ios" fails on ARM macOS, ibtoold crashes compiling xcassets/storyboards

Flutter Gallery app does not build/run on the new macOS hardware for deploying to an iOS 13.6.1 device

<details> 
<summary>doctor -v</summary> 

```

asiva@sivas-mac gallery % /Users/asiva/flutter/bin/flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel master, 1.22.0-10.0.pre.50, on macOS 11.0 20A5299w, locale en-US)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
[✓] Xcode - develop for iOS and macOS (Xcode 12.0)
[!] Android Studio (version 4.0)
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
[✓] Connected device (1 available)

! Doctor found issues in 1 category.
``` 

</details> 

```
asiva@sivas-mac gallery % /Users/asiva/flutter/bin/flutter devices  
1 connected device:

iPhone (mobile) • 00008030-001C78E13EE3802E • ios • iOS 13.6.1
```

<details> 
<summary>logs</summary> 

```
asiva@sivas-mac gallery % /Users/asiva/flutter/bin/flutter run      
Launching lib/main.dart on iPhone in debug mode...
 
Automatically signing iOS for device deployment using specified development team in Xcode project: A3HUGF6PH6
Running Xcode build...                                                  
 └─Compiling, linking and signing...                         3.6s
Xcode build done.                                           34.7s
Failed to build iOS app
Error output from Xcode build:
↳
    2020-09-02 16:16:15.703 xcodebuild[37765:3293286]  DTDeviceKit: deviceType from 00008030-001C78E13EE3802E was
    NULL
    2020-09-02 16:16:15.855 xcodebuild[37765:3293441]  DTDeviceKit: deviceType from 00008030-001C78E13EE3802E was
    NULL
    2020-09-02 16:16:15.855 xcodebuild[37765:3293441]  DTDeviceKit: deviceType from 00008030-001C78E13EE3802E was
    NULL
    ** BUILD FAILED **


Xcode's output:
↳
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:10: warning: double-quoted include
    "FlutterAppDelegate.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterAppDelegate.h"
             ^~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterAppDelegate.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegate.h:10:10: warning: double-quoted
    include "FlutterMacros.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegate.h:11:10: warning: double-quoted
    include "FlutterPlugin.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterPlugin.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterPlugin.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegate.h:11:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:11:10: warning: double-quoted include
    "FlutterBinaryMessenger.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterBinaryMessenger.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterBinaryMessenger.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegate.h:11:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:11:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterBinaryMessenger.h:10:10: warning:
    double-quoted include "FlutterMacros.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegate.h:11:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:12:10: warning: double-quoted include
    "FlutterChannels.h" in framework header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterChannels.h"
             ^~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterChannels.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegate.h:11:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:12:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterChannels.h:8:10: warning: double-quoted
    include "FlutterBinaryMessenger.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterBinaryMessenger.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterBinaryMessenger.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterChannels.h:9:10: warning: double-quoted
    include "FlutterCodecs.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterCodecs.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterCodecs.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegate.h:11:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:12:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterChannels.h:9:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterCodecs.h:9:10: warning: double-quoted include
    "FlutterMacros.h" in framework header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegate.h:11:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:13:10: warning: double-quoted include
    "FlutterCodecs.h" in framework header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterCodecs.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterCodecs.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:14:10: warning: double-quoted include
    "FlutterPlatformViews.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterPlatformViews.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterPlatformViews.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegate.h:11:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:14:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterPlatformViews.h:10:9: warning: double-quoted
    include "FlutterCodecs.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #import "FlutterCodecs.h"
            ^~~~~~~~~~~~~~~~~
            <Flutter/FlutterCodecs.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterPlatformViews.h:11:9: warning: double-quoted
    include "FlutterMacros.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #import "FlutterMacros.h"
            ^~~~~~~~~~~~~~~~~
            <Flutter/FlutterMacros.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegate.h:11:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:15:10: warning: double-quoted include
    "FlutterTexture.h" in framework header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterTexture.h"
             ^~~~~~~~~~~~~~~~~~
             <Flutter/FlutterTexture.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegate.h:11:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:15:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterTexture.h:11:10: warning: double-quoted
    include "FlutterMacros.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:9:10: warning: double-quoted include
    "FlutterBinaryMessenger.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterBinaryMessenger.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterBinaryMessenger.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:10:10: warning: double-quoted include
    "FlutterCallbackCache.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterCallbackCache.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterCallbackCache.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:10:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterCallbackCache.h:10:10: warning: double-quoted
    include "FlutterMacros.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:11:10: warning: double-quoted include
    "FlutterChannels.h" in framework header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterChannels.h"
             ^~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterChannels.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:12:10: warning: double-quoted include
    "FlutterCodecs.h" in framework header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterCodecs.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterCodecs.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:13:10: warning: double-quoted include
    "FlutterDartProject.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterDartProject.h"
             ^~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterDartProject.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:13:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterDartProject.h:10:10: warning: double-quoted
    include "FlutterMacros.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:14:10: warning: double-quoted include
    "FlutterEngine.h" in framework header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterEngine.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterEngine.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:14:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterEngine.h:11:10: warning: double-quoted include
    "FlutterBinaryMessenger.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterBinaryMessenger.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterBinaryMessenger.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterEngine.h:12:10: warning: double-quoted include
    "FlutterDartProject.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterDartProject.h"
             ^~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterDartProject.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterEngine.h:13:10: warning: double-quoted include
    "FlutterMacros.h" in framework header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterEngine.h:14:10: warning: double-quoted include
    "FlutterPlugin.h" in framework header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterPlugin.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterPlugin.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterEngine.h:15:10: warning: double-quoted include
    "FlutterTexture.h" in framework header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterTexture.h"
             ^~~~~~~~~~~~~~~~~~
             <Flutter/FlutterTexture.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:15:10: warning: double-quoted include
    "FlutterHeadlessDartRunner.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterHeadlessDartRunner.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterHeadlessDartRunner.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:15:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterHeadlessDartRunner.h:10:10: warning:
    double-quoted include "FlutterBinaryMessenger.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterBinaryMessenger.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterBinaryMessenger.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterHeadlessDartRunner.h:11:10: warning:
    double-quoted include "FlutterDartProject.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterDartProject.h"
             ^~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterDartProject.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterHeadlessDartRunner.h:12:10: warning:
    double-quoted include "FlutterEngine.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterEngine.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterEngine.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterHeadlessDartRunner.h:13:10: warning:
    double-quoted include "FlutterMacros.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:16:10: warning: double-quoted include
    "FlutterMacros.h" in framework header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:17:10: warning: double-quoted include
    "FlutterPlatformViews.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterPlatformViews.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterPlatformViews.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:18:10: warning: double-quoted include
    "FlutterPlugin.h" in framework header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterPlugin.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterPlugin.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:19:10: warning: double-quoted include
    "FlutterPluginAppLifeCycleDelegate.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterPluginAppLifeCycleDelegate.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterPluginAppLifeCycleDelegate.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:19:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterPluginAppLifeCycleDelegate.h:8:10: warning:
    double-quoted include "FlutterPlugin.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterPlugin.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterPlugin.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:20:10: warning: double-quoted include
    "FlutterTexture.h" in framework header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterTexture.h"
             ^~~~~~~~~~~~~~~~~~
             <Flutter/FlutterTexture.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:21:10: warning: double-quoted include
    "FlutterViewController.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterViewController.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterViewController.h>
    While building module 'Flutter' imported from /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/Flutter.h:21:
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterViewController.h:11:10: warning: double-quoted
    include "FlutterBinaryMessenger.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterBinaryMessenger.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterBinaryMessenger.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterViewController.h:12:10: warning: double-quoted
    include "FlutterDartProject.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterDartProject.h"
             ^~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterDartProject.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterViewController.h:13:10: warning: double-quoted
    include "FlutterEngine.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterEngine.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterEngine.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterViewController.h:14:10: warning: double-quoted
    include "FlutterMacros.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterViewController.h:15:10: warning: double-quoted
    include "FlutterPlugin.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterPlugin.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterPlugin.h>
    /Users/asiva/gallery/ios/Flutter/Flutter.framework/Headers/FlutterViewController.h:16:10: warning: double-quoted
    include "FlutterTexture.h" in framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterTexture.h"
             ^~~~~~~~~~~~~~~~~~
             <Flutter/FlutterTexture.h>
    46 warnings generated.
    While building module 'package_info' imported from
    /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.m:8:
    In file included from <module-includes>:1:
    /Users/asiva/gallery/build/ios/Debug-iphoneos/package_info/package_info.framework/Headers/package_info-umbrella.
    h:13:9: warning: double-quoted include "FLTPackageInfoPlugin.h" in framework header, expected angle-bracketed
    instead [-Wquoted-include-in-framework-header]
    #import "FLTPackageInfoPlugin.h"
            ^~~~~~~~~~~~~~~~~~~~~~~~
            <package_info/FLTPackageInfoPlugin.h>
    1 warning generated.
    While building module 'path_provider' imported from
    /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.m:14:
    In file included from <module-includes>:1:
    /Users/asiva/gallery/build/ios/Debug-iphoneos/path_provider/path_provider.framework/Headers/path_provider-umbrel
    la.h:13:9: warning: double-quoted include "FLTPathProviderPlugin.h" in framework header, expected
    angle-bracketed instead [-Wquoted-include-in-framework-header]
    #import "FLTPathProviderPlugin.h"
            ^~~~~~~~~~~~~~~~~~~~~~~~~
            <path_provider/FLTPathProviderPlugin.h>
    1 warning generated.
    While building module 'shared_preferences' imported from
    /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.m:20:
    In file included from <module-includes>:1:
    /Users/asiva/gallery/build/ios/Debug-iphoneos/shared_preferences/shared_preferences.framework/Headers/shared_pre
    ferences-umbrella.h:13:9: warning: double-quoted include "FLTSharedPreferencesPlugin.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #import "FLTSharedPreferencesPlugin.h"
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            <shared_preferences/FLTSharedPreferencesPlugin.h>
    1 warning generated.
    While building module 'url_launcher' imported from
    /Users/asiva/gallery/ios/Runner/GeneratedPluginRegistrant.m:26:
    In file included from <module-includes>:1:
    /Users/asiva/gallery/build/ios/Debug-iphoneos/url_launcher/url_launcher.framework/Headers/url_launcher-umbrella.
    h:13:9: warning: double-quoted include "FLTURLLauncherPlugin.h" in framework header, expected angle-bracketed
    instead [-Wquoted-include-in-framework-header]
    #import "FLTURLLauncherPlugin.h"
            ^~~~~~~~~~~~~~~~~~~~~~~~
            <url_launcher/FLTURLLauncherPlugin.h>
    1 warning generated.
    50 warnings generated.
    2020-09-02 16:16:48.843 ibtoold[37948:3294595] [MT] IBPlatformTool: *** Failed to launch tool with description
    <IBCocoaTouchPlatformToolDescription: 0x7f9b19f2acd0> System content for IBCocoaTouchFramework-ElevenAndLater
    <IBScaleFactorDeviceTypeDescription: 0x7f9b19f2e230> scaleFactor=2x, renderMode.identifier=(null): Failed to
    find or create execution context for description '<IBCocoaTouchPlatformToolDescription: 0x7f9b19f2acd0> System
    content for IBCocoaTouchFramework-ElevenAndLater <IBScaleFactorDeviceTypeDescription: 0x7f9b19f2e230>
    scaleFactor=2x, renderMode.identifier=(null)'.
    Device type: IBSimDeviceTypeiPad2x (com.apple.dt.Xcode.IBSimDeviceType.iPad-2x)
    Sim runtime: (null)
    ** Please also include the output of `xcrun simctl list` and `xcode-select -p`.: Failed to find newest available
    Simulator runtime: No available runtime found from: [
        watchOS 7.0 (7.0 - 18R5368d) - com.apple.CoreSimulator.SimRuntime.watchOS-7-0 (unavailable, failed to open
        liblaunch_sim.dylib) ==> not available: Error Domain=com.apple.CoreSimulator.SimError Code=401 "The watchOS
        7.0 simulator runtime is not available." UserInfo={NSLocalizedDescription=The watchOS 7.0 simulator runtime
        is not available., NSUnderlyingError=0x7f9b19b88310 {Error Domain=NSPOSIXErrorDomain Code=53 "Software
        caused connection abort" UserInfo={NSLocalizedFailureReason=XPC error talking to SimLaunchHostService:
        <dictionary: 0x7fff8ccd5a10> { count = 1, transaction: 0, voucher = 0x0, contents =
    	"XPCErrorDescription" => <string: 0x7fff8ccd5b78> { length = 18, contents = "Connection invalid" }
    }}}, NSLocalizedRecoverySuggestion=Unable to open liblaunch_sim.dylib.  Try reinstalling Xcode or the simulator
    runtime., NSLocalizedFailureReason=failed to open liblaunch_sim.dylib},
        tvOS 14.0 (14.0 - 18J5370e) - com.apple.CoreSimulator.SimRuntime.tvOS-14-0 (unavailable, failed to open
        liblaunch_sim.dylib) ==> not available: Error Domain=com.apple.CoreSimulator.SimError Code=401 "The tvOS
        14.0 simulator runtime is not available." UserInfo={NSLocalizedDescription=The tvOS 14.0 simulator runtime
        is not available., NSUnderlyingError=0x7f9b19b88310 {Error Domain=NSPOSIXErrorDomain Code=53 "Software
        caused connection abort" UserInfo={NSLocalizedFailureReason=XPC error talking to SimLaunchHostService:
        <dictionary: 0x7fff8ccd5a10> { count = 1, transaction: 0, voucher = 0x0, contents =
    	"XPCErrorDescription" => <string: 0x7fff8ccd5b78> { length = 18, contents = "Connection invalid" }
    }}}, NSLocalizedRecoverySuggestion=Unable to open liblaunch_sim.dylib.  Try reinstalling Xcode or the simulator
    runtime., NSLocalizedFailureReason=failed to open liblaunch_sim.dylib},
        iOS 14.0 (14.0 - 18A5357e) - com.apple.CoreSimulator.SimRuntime.iOS-14-0 (unavailable, failed to open
        liblaunch_sim.dylib) ==> not available: Error Domain=com.apple.CoreSimulator.SimError Code=401 "The iOS 14.0
        simulator runtime is not available." UserInfo={NSLocalizedDescription=The iOS 14.0 simulator runtime is not
        available., NSUnderlyingError=0x7f9b19b88310 {Error Domain=NSPOSIXErrorDomain Code=53 "Software caused
        connection abort" UserInfo={NSLocalizedFailureReason=XPC error talking to SimLaunchHostService: <dictionary:
        0x7fff8ccd5a10> { count = 1, transaction: 0, voucher = 0x0, contents =
    	"XPCErrorDescription" => <string: 0x7fff8ccd5b78> { length = 18, contents = "Connection invalid" }
    }}}, NSLocalizedRecoverySuggestion=Unable to open liblaunch_sim.dylib.  Try reinstalling Xcode or the simulator
    runtime., NSLocalizedFailureReason=failed to open liblaunch_sim.dylib},
    ] when matching for <IBCocoaTouchPlatformToolDescription: 0x7f9b19f2acd0> System content for
    IBCocoaTouchFramework-ElevenAndLater <IBScaleFactorDeviceTypeDescription: 0x7f9b19f2e230> scaleFactor=2x,
    renderMode.identifier=(null), idiom=<IBIPhoneIdiom: 0x7f9b19a0a7f0> runtime=<IBCocoaTouchTargetRuntime:
    0x7f9b164ac3a0>
    /* com.apple.ibtool.errors */
    /Users/asiva/gallery/ios/Runner/Base.lproj/Main.storyboard: error: Failed to find or create execution context
    for description '<IBCocoaTouchPlatformToolDescription: 0x7f9b19f2acd0> System content for
    IBCocoaTouchFramework-ElevenAndLater <IBScaleFactorDeviceTypeDescription: 0x7f9b19f2e230> scaleFactor=2x,
    renderMode.identifier=(null)'.
    Device type: IBSimDeviceTypeiPad2x (com.apple.dt.Xcode.IBSimDeviceType.iPad-2x)
    Sim runtime: (null)
    ** Please also include the output of `xcrun simctl list` and `xcode-select -p`.
        Underlying Errors:
            Description: Failed to find newest available Simulator runtime
            Underlying Errors:
                Description: No available runtime found from: [
        watchOS 7.0 (7.0 - 18R5368d) - com.apple.CoreSimulator.SimRuntime.watchOS-7-0 (unavailable, failed to open
        liblaunch_sim.dylib) ==> not available: Error Domain=com.apple.CoreSimulator.SimError Code=401 "The watchOS
        7.0 simulator runtime is not available." UserInfo={NSLocalizedDescription=The watchOS 7.0 simulator runtime
        is not available., NSUnderlyingError=0x7f9b19b88310 {Error Domain=NSPOSIXErrorDomain Code=53 "Software
        caused connection abort" UserInfo={NSLocalizedFailureReason=XPC error talking to SimLaunchHostService:
        <dictionary: 0x7fff8ccd5a10> { count = 1, transaction: 0, voucher = 0x0, contents =
    	"XPCErrorDescription" => <string: 0x7fff8ccd5b78> { length = 18, contents = "Connection invalid" }
    }}}, NSLocalizedRecoverySuggestion=Unable to open liblaunch_sim.dylib.  Try reinstalling Xcode or the simulator
    runtime., NSLocalizedFailureReason=failed to open liblaunch_sim.dylib},
        tvOS 14.0 (14.0 - 18J5370e) - com.apple.CoreSimulator.SimRuntime.tvOS-14-0 (unavailable, failed to open
        liblaunch_sim.dylib) ==> not available: Error Domain=com.apple.CoreSimulator.SimError Code=401 "The tvOS
        14.0 simulator runtime is not available." UserInfo={NSLocalizedDescription=The tvOS 14.0 simulator runtime
        is not available., NSUnderlyingError=0x7f9b19b88310 {Error Domain=NSPOSIXErrorDomain Code=53 "Software
        caused connection abort" UserInfo={NSLocalizedFailureReason=XPC error talking to SimLaunchHostService:
        <dictionary: 0x7fff8ccd5a10> { count = 1, transaction: 0, voucher = 0x0, contents =
    	"XPCErrorDescription" => <string: 0x7fff8ccd5b78> { length = 18, contents = "Connection invalid" }
    }}}, NSLocalizedRecoverySuggestion=Unable to open liblaunch_sim.dylib.  Try reinstalling Xcode or the simulator
    runtime., NSLocalizedFailureReason=failed to open liblaunch_sim.dylib},
        iOS 14.0 (14.0 - 18A5357e) - com.apple.CoreSimulator.SimRuntime.iOS-14-0 (unavailable, failed to open
        liblaunch_sim.dylib) ==> not available: Error Domain=com.apple.CoreSimulator.SimError Code=401 "The iOS 14.0
        simulator runtime is not available." UserInfo={NSLocalizedDescription=The iOS 14.0 simulator runtime is not
        available., NSUnderlyingError=0x7f9b19b88310 {Error Domain=NSPOSIXErrorDomain Code=53 "Software caused
        connection abort" UserInfo={NSLocalizedFailureReason=XPC error talking to SimLaunchHostService: <dictionary:
        0x7fff8ccd5a10> { count = 1, transaction: 0, voucher = 0x0, contents =
    	"XPCErrorDescription" => <string: 0x7fff8ccd5b78> { length = 18, contents = "Connection invalid" }
    }}}, NSLocalizedRecoverySuggestion=Unable to open liblaunch_sim.dylib.  Try reinstalling Xcode or the simulator
    runtime., NSLocalizedFailureReason=failed to open liblaunch_sim.dylib},
    ] when matching for <IBCocoaTouchPlatformToolDescription: 0x7f9b19f2acd0> System content for
    IBCocoaTouchFramework-ElevenAndLater <IBScaleFactorDeviceTypeDescription: 0x7f9b19f2e230> scaleFactor=2x,
    renderMode.identifier=(null), idiom=<IBIPhoneIdiom: 0x7f9b19a0a7f0> runtime=<IBCocoaTouchTargetRuntime:
    0x7f9b164ac3a0>
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported
    deployment target versions is 9.0 to 14.0.99. (in target 'url_launcher' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported
    deployment target versions is 9.0 to 14.0.99. (in target 'shared_preferences' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported
    deployment target versions is 9.0 to 14.0.99. (in target 'path_provider' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported
    deployment target versions is 9.0 to 14.0.99. (in target 'package_info' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported
    deployment target versions is 9.0 to 14.0.99. (in target 'Flutter' from project 'Pods')

Could not build the precompiled application for the device.

Error launching application on iPhone.

``` 

</details>

--- a/packages/flutter_tools/lib/src/macos/xcode.dart
+++ b/packages/flutter_tools/lib/src/macos/xcode.dart
@@ -174,7 +174,7 @@ class Xcode {
   /// tools and properties.
   ///
   /// Returns `xcrun` on x86 macOS.
-  /// Returns `/usr/bin/arch -arm64 xcrun` on ARM macOS to force Xcode commands
+  /// Returns `/usr/bin/arch -arm64e xcrun` on ARM macOS to force Xcode commands
   /// to run outside the x86 Rosetta translation, which may cause crashes.
   List<String> xcrunCommand() {
     final List<String> xcrunCommand = <String>[];
@@ -182,7 +182,7 @@ class Xcode {
       // Force Xcode commands to run outside Rosetta.
       xcrunCommand.addAll(<String>[
         '/usr/bin/arch',
-        '-arm64',
+        '-arm64e',
       ]);
     }
     xcrunCommand.add('xcrun');

--- a/packages/flutter_tools/test/general.shard/macos/xcode_test.dart
+++ b/packages/flutter_tools/test/general.shard/macos/xcode_test.dart
@@ -265,7 +265,7 @@ void main() {
 
           expect(xcode.xcrunCommand(), <String>[
             '/usr/bin/arch',
-            '-arm64',
+            '-arm64e',
             'xcrun',
           ]);
           expect(fakeProcessManager.hasRemainingExpectations, isFalse);

